ngraph/test/files/paddlepaddle/gen_scripts/generate_rnn_lstm.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+from save_model import saveModel
+import sys
+
+
+def pdpd_rnn_lstm(input_size, hidden_size, layers, direction):
+    import paddle as pdpd
+    pdpd.enable_static()
+    main_program = pdpd.static.Program()
+    startup_program = pdpd.static.Program()
+
+    num_of_directions = 1 if direction == 'forward' else 2
+    with pdpd.static.program_guard(main_program, startup_program):
+
+        rnn = pdpd.nn.LSTM(input_size, hidden_size, layers, direction)
+
+        data = pdpd.static.data(name='x', shape=[4, 3, input_size], dtype='float32')
+        prev_h = pdpd.ones(shape=[layers * num_of_directions, 4, hidden_size], dtype=np.float32)
+        prev_c = pdpd.ones(shape=[layers * num_of_directions, 4, hidden_size], dtype=np.float32)
+
+        y, (h, c) = rnn(data, (prev_h, prev_c))
+
+        cpu = pdpd.static.cpu_places(1)
+        exe = pdpd.static.Executor(cpu[0])
+        exe.run(startup_program)
+
+        outs = exe.run(
+            feed={'x': np.ones([4, 3, input_size]).astype(np.float32)},
+            fetch_list=[y, h, c],
+            program=main_program)
+        saveModel("rnn_lstm_layer_" + str(layers) + '_' + str(direction), exe, feedkeys=['x'],
+                  fetchlist=[y, h, c], inputs=[np.ones([4, 3, input_size]).astype(np.float32)], outputs=[outs[0], outs[1], outs[2]], target_dir=sys.argv[1])
```

Comment:
@nosovmik To inspect the hidden states of LSTMCells, I save the updated hidden states of PDPD model and then compare it with the OV's updated hidden states. The comparing results shows that hidden states are updated correctly by OV LSTMSqeuence cells. Therefore I think just one-time inference, we can still check whether the hidden states are updated correctly.